README.md

```diff
@@ -10,38 +10,39 @@ This repository hosts a number of [Kotlin](https://kotlinlang.org) wrappers for
+| [kotlin-csstype](kotlin-csstype/README.md)                         | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-csstype)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-csstype)                          | 
+| [kotlin-emotion](kotlin-emotion/README.md)                         | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-emotion)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-emotion)                          | 
+| [kotlin-extensions](kotlin-extensions/README.md)                   | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-extensions)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-extensions)                    | 
+| [kotlin-history](kotlin-history/README.md)                         | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-history)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-history)                          | 
+| [kotlin-mui](kotlin-mui/README.md)                                 | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-mui)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-mui)                                  | 
+| [kotlin-mui-icons](kotlin-mui-icons/README.md)                     | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-mui-icons)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-mui-icons)                      | 
+| [kotlin-node](kotlin-node/README.md)                               | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-node)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-node)                                |
+| [kotlin-popper](kotlin-popper/README.md)                           | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-popper)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-popper)                            | 
+| [kotlin-react](kotlin-react/README.md)                             | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-react)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-react)                              | 
+| [kotlin-react-beautiful-dnd](kotlin-react-beautiful-dnd/README.md) | [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-react-beautiful-dnd)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-react-beautiful-dnd)  | 
```

Comment:
The new line with `kotlin-react-beautiful-dnd` became the longest, so autoformatting moved other lines too. Let me know if I should revert it.